app/src/main/res/values-ar/strings.xml

```diff
@@ -8,159 +8,145 @@
-  <string name="stop_exploration_dialog_cancel_button">إلغاء</string>
-  <string name="stop_exploration_dialog_leave_button">مغادرة</string>
-  <string name="exploration_exit_button_spotlight_hint">اخرج في أي وقت باستخدام هذا الزر. سنحفظ تقدمك.</string>
-  <string name="voiceover_icon_spotlight_hint">هل ترغب في أن يقرأ %s لك؟ اضغط على هذا الزر لتجربته!</string>
-  <string name="voiceover_language_icon_spotlight_hint">انقر لتغيير لغة التعليق الصوتي.</string>
-  <string name="progress_database_full_dialog_title">تم الوصول إلى الحد الأقصى من سعة التخزين</string>
-  <string name="progress_database_full_dialog_description">سوف يتم حذف تقدمك في درس %s</string>
-  <string name="progress_database_full_dialog_continue_button">استمرار</string>
-  <string name="progress_database_full_dialog_leave_without_saving_progress_button">المغادرة بدون حفظ التقدم</string>
-  <string name="progress_database_full_dialog_back_to_lesson_button">العودة إلى الدرس</string>
-  <string name="topic_practice_master_these_skills">أتقن هذه المهارات</string>
-  <string name="topic_name_text_view">النسب والاستدلال النسبي</string>
-  <string name="topic_practice_skills_description">قم باختيار المهارات التي تريد أن تتدرب عليها.</string>
-  <string name="topic_practice_start">ابدأ</string>
-  <string name="info_tab_content_description">تفاصيل حول الموضوع.</string>
-  <string name="revision_tab_content_description">راجع المفاهيم المدروسة من الدروس المكتملة هنا.</string>
-  <string name="practice_tab_content_description">تدرب على المفاهيم المكتسبة من الدروس المكتملة.</string>
-  <string name="lessons_tab_content_description">انقر هنا لبدء تشغيل درس.</string>
-  <string name="topic_lessons_tab_spotlight_hint">ابحث عن كل دروسك هنا.</string>
-  <string name="topic_revision_tab_spotlight_hint">راجع المفاهيم المكتسبة من الدروس المكتملة هنا.</string>
-  <string name="first_chapter_spotlight_hint">انقر هنا لبدء تشغيل درس.</string>
-  <string name="state_continue_button">استمرار</string>
-  <string name="state_continue_submit">تقديم</string>
-  <string name="previous_state_description">التوجه إلى البطاقة السابقة</string>
-  <string name="next_state_description">التوجه إلى البطاقة القادمة</string>
-  <string name="state_submit_button">تقديم</string>
-  <string name="lesson_replay_button_text">إعادة التشغيل</string>
-  <string name="state_end_exploration_button">العودة إلى الموضوع</string>
-  <string name="previous_responses_header">الردود السابقة (%s)</string>
-  <string name="image_interaction_answer_text">يضغط على %s</string>
-  <string name="state_learn_again_button">تعلم مرة أخرى</string>
-  <string name="see_more">اعرض المزيد</string>
-  <string name="see_less">اعرض أقل</string>
-  <string name="FAQs">الأسئلة المتكررة</string>
-  <string name="featured_questions">الأسئلة المميزة</string>
-  <string name="frequently_asked_questions">الأسئلة المتكررة</string>
+  <string name="stop_exploration_dialog_cancel_button_text">إلغاء</string>
```

Comment:
Should we not also organize translations by categories like in the English file?

Reply:
@adhiamboperes no I think as per Ben we can skip translation files for now?